purego/embed/vendor_linux_amd64.go

```diff
@@ -0,0 +1,22 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+// Package vendor is required to help go tools support vendoring.
+// DO NOT REMOVE
+
+//go:build linux && amd64
+
+package vendor
+
+// Adds a dynamic import for libm.so because libddwaf needs the cielf symbol
+//go:cgo_import_dynamic purego_ceilf ceilf "libm.so.6"
+//go:cgo_import_dynamic _ _ "libm.so.6"
+
+import _ "embed"
+
+//go:embed linux-amd64
+var Libddwaf []byte
+
+var purego_cielf uint
```

Comment:
\n-terminated file, or not? ^^

Reply:
fixed

Reply:
Removed all '\n'

Reply:
I prefer them, but no strong opinions ;-p
IIRC, it was enforced in my EPITA coding style at my time ;-p